services/src/main/java/io/druid/cli/CliPeon.java

```diff
@@ -243,7 +242,7 @@ public void configure(Binder binder)
             // configuration of other parameters, but I don't think that's actually a problem.
             // Note, if that is actually not a problem, then that probably means we have the wrong abstraction.
             binder.bind(SegmentLoaderConfig.class)
-                  .toInstance(new SegmentLoaderConfig().withLocations(Arrays.<StorageLocationConfig>asList()));
+                  .toInstance(new SegmentLoaderConfig().withLocations(Collections.emptyList()));
```

Comment:
If you wish you could also fix all occurrences of this, via IntelliJ inspection "Too few arguments of Arrays.asList()", and prohibit it on CI level. There are hundreds of such issues in the codebase, but since there is an automatic fix in IntelliJ, it could be even not very hard to do this.

Reply:
Okay, I will add this situation to the Intellij inspection configuration file. In this way, each newly added developer will notice this coding specification without having to read some sort of **HOW TO CONTRIBUTE** documents. :+1:

Reply:
Already add `ArraysAsListWithZeroOrOneArgument` specification into profile.